### PR TITLE
Obmib 322 create product owner data

### PR DESCRIPTION
**Pull request by Kyslytsia**
Create ProductOwnerData

### CODE REVIEW

File: src/components/product-owner-data/index.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const ProductOwnerData = ({
+  avatar,
+  name,
+  date,
+  city,
+  phone,
+  translatedTextDate,
+  translatedTextCity,
+  translatedTextPhone,
+}: IProductOwnerDataProps) => {
```
Comment (Voloshkina) [inline src/components/product-owner-data/index.tsx:15]: sort pls

File: src/components/product-owner-data/arg-types.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  },
+  phone: {
+    name: 'phone',
+    type: { name: 'string' },
+    description: 'phone',
+    table: {
+      type: { summary: 'string' },
+    },
+  },
+  translatedText: {
```
Comment (Voloshkina) [inline src/components/product-owner-data/arg-types.ts:42]: maybe you don't need translatedText?

File: src/components/product-owner-data/index.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import * as Styles from './styles';
+import { Avatar } from '../avatar';
+import { EllipsisText } from '../ellipsis-text';
+import { IProductOwnerDataProps } from './types';
+
+const ProductOwnerData = ({
+  avatar,
+  name,
+  date,
+  city,
```
Comment (Voloshkina) [inline src/components/product-owner-data/index.tsx:10]: sort pls

File: src/components/product-owner-data/index.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+                {translatedTextPhone}
+              </Styles.PostReqDataText>
+            )}
+          </Styles.PostReqDataBlock>
+        </Styles.PostReqData>
+
+        <Styles.PostResData>
+          <Styles.PostReqDataBlock>
+            {date && (
+              <Styles.PostReqDataText style={{ color: 'black' }}>
```
Comment (Wolshebnik) [inline src/components/product-owner-data/index.tsx:52]: you need to use color from theme

File: src/components/product-owner-data/index.tsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                {date}
+              </Styles.PostReqDataText>
+            )}
+
+            <Styles.PostReqDataText style={{ width: '200px', color: 'black' }}>
```
Comment (Wolshebnik) [inline src/components/product-owner-data/index.tsx:57]: the same

File: src/components/product-owner-data/styles.ts [hunk 6]
```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  background-color: #fdfcff;
```
Comment (Wolshebnik) [inline src/components/product-owner-data/styles.ts:4]: use background color from theme

File: src/components/product-owner-data/styles.ts [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+export const PostReqDataBlock = styled.div`
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 26px;
+`;
+
+export const PostReqDataText = styled.div`
+  color: #8f8f8f;
```
Comment (Wolshebnik) [inline src/components/product-owner-data/styles.ts:52]: color from theme

File: src/components/product-owner-data/index.tsx [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+              <EllipsisText id={city} width={300}>
+                {city}
+              </EllipsisText>
+            </Styles.PostReqDataText>
+
+            {phone && (
+              <Styles.PostReqDataText style={{ color: 'black' }}>
```
Comment (Wolshebnik) [inline src/components/product-owner-data/index.tsx:64]: the same

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

**Review by Voloshkina — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

File: src/components/product-owner-data/styles.ts [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+  width: 170px;
+  padding: 5px 20px 15px 30px;
+`;
+
+export const PostResData = styled.div`
+  padding: 5px 20px 15px 30px;
+`;
+
+export const PostReqDataBlock = styled.div`
+  font-family: 'Roboto', sans-serif;
```
Comment (Wolshebnik) [inline src/components/product-owner-data/styles.ts:44]: Roboto is the default font
check pls
if true remove

File: src/components/product-owner-data/types.ts [hunk 10]
```diff
@@ -0,0 +1,10 @@
+export interface IProductOwnerDataProps {
+  name: string;
+  date?: string;
+  city: string;
+  phone?: string;
+  avatar: string;
```
Comment (Wolshebnik) [inline src/components/product-owner-data/types.ts:6]: avatar should be optional

File: src/config/theme/colors.ts [hunk 11]
```diff
@@ -60,6 +62,11 @@ const itemCard = Object.freeze({
   borderCard: silver,
 });
 
+const productOwnerData = Object.freeze({
+  ownerDataBG: magnolia,
```
Comment (Wolshebnik) [inline src/config/theme/colors.ts:66]: name choice is weird
just bg

File: src/config/theme/colors.ts [hunk 12]
```diff
@@ -60,6 +62,11 @@ const itemCard = Object.freeze({
...
+  ownerDataText: grey,
```
Comment (Wolshebnik) [inline src/config/theme/colors.ts:67]: name choice is weird
just text

File: src/components/product-owner-data/styles.ts [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+export const ProductOwnerAvatar = styled.div`
+  padding: 35px 15px 20px 35px;
+`;
+
+export const ProductOwnerName = styled.div`
+  padding-top: 30px;
+`;
+
+export const ProductOwnerTitleH2 = styled.h2`
+  font-family: 'Roboto', sans-serif;
```
Comment (Wolshebnik) [inline src/components/product-owner-data/styles.ts:23]: Roboto is the default font
check pls
if true remove

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

**Review by Viktoria-Serkiz — APPROVED** [review]

File: src/components/product-owner-data/styles.ts [hunk 14]
```diff
@@ -0,0 +1,55 @@
+import styled, { css } from 'styled-components';
+
+export const Container = styled.div`
+  background-color: ${({ theme }) => theme.colors.productOwnerData.bg};
+  padding-bottom: 15px;
```
Comment (cloudmenson) [inline src/components/product-owner-data/styles.ts:5]: The background-color property must be lower than the padding

File: src/components/index.tsx [hunk 15]
```diff
@@ -12,4 +12,5 @@ export { PagePagination } from './page-pagination';
 export { ProductPostData } from './product-post-data';
 export { Title } from './title';
 export { Toast, showMessage } from './toast';
+export {ProductOwnerData} from './product-owner-data'
```
Comment (cloudmenson) [inline src/components/index.tsx:15]: Use prettier: { ProductOwnerData }

**Review by AkintsevaElizaveta — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

**Pull request merged**